config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] [OPTIONAL]
To ensure that we don't expose unauthorized routes/pages let ensure to put restrictions on the pages we don't want to be exposed by doing something like:

```ruby
# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```